src/bootstrap/bootstrap.py

```diff
@@ -732,19 +732,27 @@ def build_bootstrap(self, color):
             (os.pathsep + env["LIBRARY_PATH"]) \
             if "LIBRARY_PATH" in env else ""
 
+        # Export Stage0 snapshot compiler related env variables
+        build_section = f"target.{self.build}"
+        if self.build != None:
```

Comment:
This will always be true, we use it to download the beta compiler. So you can remove the `of` and unindent this whole block.

Reply:
Done